code/src/components/Form.js

```diff
@@ -0,0 +1,115 @@
+import React, {useState} from 'react'
+
+import OrderBtn from './OrderBtn'
+import SelectSize from './SelectSize'
+import Name from './Name'
+import Email from './Email'
+import Hero from './HeroImage'
+import Phone from './Phone'
+import Amount from './Amount'
+import Checkbox from './Checkbox'
+import Radio from './Radio'
+import Range from './RangeSlider'
+
+const Form = () => {
+    const [sumHidden, setsumHidden] = useState(true);
+    const [Size, setSize] = useState("");
+    const [myamount, setmyamount] = useState("");
+    const [Color, setColor] = useState();
+    const [myname, setmyname] = useState("");
+    const [myemail, setmyemail] = useState("");
+    const [wantsNewsletter, setwantsNewsletter] = useState("");
+    const [myphone, setmyphone] = useState("");
+    const [Rng, setRng] = useState("No Opinion");
+
```

Comment:
I see that you named some of the states with a capital letter, like Size and Color. Maybe it is better to be consistent and use lowercase.